tests_e2e/tests/scripts/agent-service

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
```

Comment:
could you update install-agent to use this script?

Reply:
ok, please add a TODO comment here. my next PR will have the changes to copy these tools to the test VM and then you can address the TODO